backends/qualcomm/utils/utils.py

```diff
@@ -93,6 +93,12 @@
 from torch.fx import passes
 from torch.fx.passes.operator_support import OperatorSupportBase
 from torch.library import Library
+from graphviz import Digraph
```

Comment:
Could you create a `debugger` folder under `backends/qualcomm` and move this implementation to there. Then we could get rid of dependency of python package in `utils.py`.

Reply:
The above modifications have been completed. Please help confirm.